workflow/rocoto/gefs_xml.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+from rocoto.workflow_xml import RocotoXML
+from applications.applications import AppConfig
+from pygw.timetools import to_timedelta
+from typing import Dict
+
+
+# Copy of GFSForecastOnlyRocotoXML for now, other than changing cycledef names from 'gfs' to 'gefs'
+#   If it remains this way, we can consolidate into a single forecast-only class
+class GEFSRocotoXML(RocotoXML):
+
+    def __init__(self, app_config: AppConfig, rocoto_config: Dict) -> None:
+        super().__init__(app_config, rocoto_config)
+
+    def _get_cycledefs(self):
```

Comment:
This shouldn't be `_get_cycledefs` any more as it is a concrete implementation *and* no longer a private